### ISSUE TITLE
No windows controls on linux

### ISSUE THREAD

**Issue by floriangosse** [Soundnode/soundnode-app#367]
I use GNOME Shell 3.10.4 on Ubuntu 14.04

![Screenshot - Linux no window controls](https://cloud.githubusercontent.com/assets/3819038/9881628/173a979e-5bd1-11e5-93e4-7093088d2bc4.png)

**Comment by Pitros** [comment]
It's x64 or x86?
Could you press `CTRL + /` it should open dev tools window, then go to console and type `navigator.platform` and please write what it returns.

**Comment by floriangosse** [comment]
```
> console.log(navigator.platform)
Linux x86_64
```

**Comment by weblancaster** [comment]
@floriangosse can you actually do the same steps but console `process.platform` please?

**Comment by floriangosse** [comment]
@weblancaster:
```
> console.log(process.platform)
linux
```

**Comment by floriangosse** [comment]
@weblancaster We need an additional check for simple `linux` at line https://github.com/Soundnode/soundnode-app/blob/ebf617204e34f83b192495d4396592ef75e31dbf/app/public/js/common/appCtrl.js#L9

**Comment by weblancaster** [comment]
yep.

**Comment by weblancaster** [comment]
I fixed.. and going to push with another fix soon.

**Comment by floriangosse** [comment]
Thx

**Comment by weblancaster** [comment]
resolved on #368 merge
